airbyte-integrations/connectors-performance/destination-harness/src/main/java/io/airbyte/integrations/destination_performance/Main.java

```diff
@@ -26,16 +28,23 @@ public class Main {
   private static final String CREDENTIALS_PATH = "secrets/%s_%s_credentials.json";
 
   public static void main(final String[] args) {
+    // If updating args for Github Actions, also update the run-performance-test.yml file
     log.info("args: {}", Arrays.toString(args));
     String image = null;
     String dataset = "1m";
+    int numOfParallelStreams = 1;
 
     switch (args.length) {
       case 1 -> image = args[0];
       case 2 -> {
         image = args[0];
         dataset = args[1];
       }
+      case 3 -> {
```

Comment:
Can there be a case of just connector and num_streams?
i.e no dataset

Reply:
Good catch, I'll update

Reply:
Actually, look into this further, there's a default value for the GitHub Actions so really this wouldn't work consistently if you were running locally and in your own `harness.yaml` it didn't have a dataset

Reply:
OK. because gha itself has a default value set for `dataset`